README.md

```diff
@@ -35,219 +31,101 @@ The SDK simplifies the setup and interaction with SAP AI Core, allowing you to f
+var config = new OrchestrationModuleConfig()
+        .withLlmConfig(OrchestrationAiModel.GPT_4O);
 ```
 
-</details>
+Now we can create our first prompt:
 
-## Getting Started
-
-### What You'll Build
-
-In this quickstart, you'll build a simple Spring Boot application that uses the OpenAI GPT-3.5 Turbo model for chat completion.
-The application will send a prompt to the AI model and display the generated response.
-
-### Prerequisites
-
-Before you begin, ensure you have:
-
-- Met the [General Requirements](#general-requirements).
-- Met the OpenAI Chat Completion module specific requirements
-    - Refer to [Prerequisites for OpenAI Chat Completion](docs/guides/OPENAI_CHAT_COMPLETION.md#prerequisites)
-- Set up the AI Core credentials
-  using [(1) Environment variable or env-file](#option-1-set-ai-core-credentials)
-  or [(2) Regular Service Binding](#option-2-regular-service-binding-in-sap-btp-cloud-foundry).
-- Deployed the OpenAI GPT-3.5 Turbo model in SAP AI Core.
-    - Refer to [Deploying the OpenAI GPT-3.5 Turbo Model](docs/guides/OPENAI_CHAT_COMPLETION.md#usage)
+```java
+var prompt = new OrchestrationPrompt("Hello world! Why is this phrase so famous?");
 
-### Write the Code
+var result = client.chatCompletion(prompt, config).getContent();
+```
 
-Add the following code to the controller or service class in your Spring Boot application:
+The result will be the text generated by the AI model.
 
-```java
-// Initialize the client for GPT-3.5 Turbo model
-OpenAiClient client = OpenAiClient.forModel(OpenAiModel.GPT_35_TURBO);
-
-// Perform chat completion
-OpenAiChatCompletionOutput result =
-    client
-        .withSystemPrompt("You are a helpful assistant.")
-        .chatCompletion("Hello World! Why is this phrase so famous?");
-```
+### Run the Application Locally
 
-### Run and Test the Application Locally
+In order to run the application locally, you need to provide credentials for your AI Core service instance.
 
-Then, compile and run your application:
+For this example we'll use a **service key** and pass it as an **environment variable** to the application.
 
 ```shell
 cd your-spring-app/
 
-mvn compile
+# assuming a bash, for other shells (e.g. PowerShell) see the below documentation
+export AICORE_SERVICE_KEY='{ "clientid": "...", "clientsecret": "...", "url": "...", "serviceurls": { "AI_API_URL": "..." } }'
+
+# assuming Maven and a Spring Boot application
 mvn spring-boot:run
 ```
 
-### Deploying to Cloud Foundry (Optional)
-
-When deploying your productive application to Cloud Foundry, it is recommended to use **service binding** to provide the AI Core credentials as per [Option 2](#option-2-regular-service-binding-in-sap-btp-cloud-foundry).
-
-Build your application using Maven and deploy it to Cloud Foundry:
+Please find **detailed instructions** and more examples [in this documentation](docs/guides/CONNECTING_TO_AICORE.md#using-the-aicore_service_key-environment-variable).
 
- ```shell
- cf push
```

Comment:
This section got removed, because it only works if the user has manually set up a CF app, which wasn't explained in this tutorial. We could also add it back in, but then I'd refer to our sample code manifest file as an example..